L1Trigger/TrackFindingTracklet/test/L1TrackNtupleMaker_cfg.py

```diff
@@ -148,6 +148,22 @@
     L1TRK_LABEL = "Level1TTTracks"
     L1TRUTH_NAME = "TTTrackAssociatorFromPixelDigisExtended"
 
+# HYBRID_NEWKF: prompt tracking
+elif (L1TRKALGO == 'HYBRID_NEWKF'):
+    process.load( 'L1Trigger.TrackFindingTracklet.ProducerKF_cff' )
+    NHELIXPAR = 4
+    L1TRK_NAME  = process.TrackFindingTrackletProducerKF_params.LabelTT.value()
```

Comment:
As the KFout module now produces tracks, I'd expect we need to change L1TRK_NAME to use "labelKFout" instead of "labelTT", to ensure the L1TrackNtuplePlot.cc tool runs on the KFout EDProduct. However, doing this crashes with error:

Principal::getByToken: Found zero products matching all criteria
Looking for type: std::vector<TTTrack<edm::Ref<edm::DetSetVector<Phase2TrackerDigi>,Phase2TrackerDigi,edm::refhelper::FindForDetSetVector<Phase2TrackerDigi> > > >
Looking for module label: TrackFindingTrackletProducerKFout
Looking for productInstanceName: TrackAccepted